### PR TITLE
Spanish Version

### PR DESCRIPTION
**Pull request by SuperAtic**
Salvando Satoshi

## Pull Request checklist

Please review the below PR requirements:

- Build was run locally and any changes were pushed
- Ensure that the title clearly describes the changes
- Issue to be referenced in the PR description rather than in the commits or PR title
- Clean commit history

### CODE REVIEW

**Comment by ecurrencyhodler** [comment]
Closes #361

File: i18n/locales/en.ts
Comment (afterburn) [inline i18n/locales/en.ts:64]: Please add your social to the following code in `/app/[lang]/about/page.tsx` instead.

```ts
const CONTRIBUTORS = [
  { username: 'adamcjonas', url: 'https://twitter.com/adamcjonas' },
  { username: 'afterburn', url: 'https://github.com/afterburn' },
  { username: 'benalleng', url: 'https://github.com/benalleng' },
  { username: 'ecurrencyhodler', url: 'https://twitter.com/ecurrencyhodler' },
  { username: 'gbks', url: 'https://twitter.com/gbks' },
  { username: 'satsie', url: 'https://twitter.com/satsie' },
  { username: 'shaavan', url: 'https://github.com/shaavan' },
  { username: 'vinayaksh42', url: 'https://github.com/vinayaksh42' },
]
```

File: i18n/locales/en.ts
Comment (afterburn) [inline i18n/locales/en.ts:130]: The backtices were there for a reason since this doesn't compile. Please run `yarn build`, it should fail with this change and give you a sense of where you need to revert some changes.

**Review by afterburn — CHANGES_REQUESTED** [review]
Thanks for your contribution @SuperAtic, I requested some changes before we can merge your PR.

File: i18n/locales/es.ts
Comment (jmunoz94) [inline i18n/locales/es.ts:61]:
```suggestion
      title: '¡Disfrútalo!',
```

File: i18n/locales/es.ts
Comment (jmunoz94) [inline i18n/locales/es.ts:79]:
```suggestion
      paragraph_one: 'Unos instantes después, tu pantalla se activa.',
```

File: i18n/locales/es.ts
Comment (jmunoz94) [inline i18n/locales/es.ts:80]: Bitcóin - bitcoines ([reference from Fundeú](https://www.fundeu.es/recomendacion/bitcoin-bitcoins-bitcoines/)). This one also applies to other places, but I won't spam the PR highlighting them all.

```suggestion
      paragraph_two: '-Deborah Chunk: "Thomas Vanderpoole. Como CEO de BitRey, diriges uno de los mayores pools de minería de bitcoines del mundo. También fabricas máquinas para minar bitcóin. ¿Qué está ocurriendo? ¿Está muriendo bitcoin?”',
```

File: i18n/locales/es.ts
Comment (jmunoz94) [inline i18n/locales/es.ts:88]: I did not even know what a "boop" is, but according to [this reddit post](https://www.reddit.com/r/learnspanish/comments/mlbi0q/boop/) there is no translation for that. There they suggest using the same "boop" (or its Spanish adaptation "bup"), but I have never in my life heard anyone call that a "boop" in Spanish. I think putting it in terms of "golpear" (hit) would keep the meaning closer to the original (technically "dándole un golpecito en" would be better, but that would make the text too long). "Abuchear" is to boo, so it makes no sense in this context:

```suggestion
        'Recibes un holocat de alguien que usa el nombre de Satoshi Nakamoto. (Es como cualquier otro e-holograma, pero este tiene forma de gato.) Abres el holocat golpeando su nariz.',
```

File: i18n/locales/es.ts
Comment (jmunoz94) [inline i18n/locales/es.ts:113]:
```suggestion
        'Desplázate hacia abajo y expande los detalles de la única transacción que se almacena en este bloque. Busca la entrada llamada "Coinbase". Ahora busca la etiqueta "SCRIPTSIG (<Tooltip id="genesis_two_paragraph_two" content="genesis_two.tooltip_hex" className="underline">HEX</Tooltip>)". El valor que aparece junto a él es un mensaje codificado',
```

File: i18n/locales/es.ts
Comment (jmunoz94) [inline i18n/locales/es.ts:116]:
```suggestion
        'Un <a href="https://bitcoinops.org/en/topics/block-explorers/" target="_blank" rel="noreferrer">explorador de bloques</a> es una herramienta útil para buscar rápidamente información sobre transacciones bitcoin.',
```

File: i18n/locales/es.ts
Comment (jmunoz94) [inline i18n/locales/es.ts:114]:
```suggestion
      paragraph_three: 'Copia este valor y pégalo aquí abajo.',
```

File: i18n/locales/es.ts
Comment (jmunoz94) [inline i18n/locales/es.ts:127]:
```suggestion
        'El mensaje que encontraste estaba codificado en un formato llamado HEX. Ahora ejecutaremos un comando para convertirlo en ASCII, el cual podremos leer',
```

File: i18n/locales/es.ts
Comment (jmunoz94) [inline i18n/locales/es.ts:130]: Random use of "usted" after having used "tú' all along. You should also be consistent with "Enter" (used in the line above) and "Intro"

```suggestion
      terminal_challenge_lines: 'Introduce tus comandos aquí y pulsa Intro...\n\n La variable $scriptSigHex ya está definida para ti.\n\n var $scriptSigHex = "04fff...e6b73"',
```

File: i18n/locales/es.ts
Comment (jmunoz94) [inline i18n/locales/es.ts:132]: My guess is that the thing to the left of the colon is some kind of key and should not be translated. Missing opening exclamation mark:

```suggestion
      success: '¡Buen trabajo! El mensaje descifrado hace referencia a la portada de <Link href="https://en.bitcoin.it/wiki/Genesis_block" className="underline">The Times</Link> del 3 de enero de 2009, el mismo día que Satoshi minó el bloque génesis. ¿No es genial? Este mensaje también nos da una idea de su motivación para crear bitcoin,',
```

File: i18n/locales/es.ts
Comment (jmunoz94) [inline i18n/locales/es.ts:143]: This one is slightly hard to translate, but I think "negociar" is closer to the original meaning than "transaccionar" (which seems to only have a political context):

```suggestion
      title: 'Negociando',
```

A compound term could be an even better translation, but IDK if it would be acceptable given the extra size:

```suggestion
      title: 'Ejecutando transacciones',
```

File: i18n/locales/es.ts
Comment (jmunoz94) [inline i18n/locales/es.ts:157]:
```suggestion
        '1. Haz clic <Link href="https://blockstream.info/tx/ff9148605a772a51cba39004df5fb042d40515967a3e38ff5294cfd017c452a9?expand" className="underline">aquí</Link> para ver una transacción específica.',
```

File: i18n/locales/es.ts
Comment (jmunoz94) [inline i18n/locales/es.ts:159]: Using "usted" instead of "tú" (which is what you have mostly used):

```suggestion
        '2. Abre los detalles y busca la parte que es de tipo "OP_RETURN".',
```

File: i18n/locales/es.ts
Comment (jmunoz94) [inline i18n/locales/es.ts:161]:
```suggestion
        '3. Ahora encuentra el campo "SCRIPTPUBKEY (ASM)". ¿Ves la parte "OP_RETURN OP_PUSHBYTES_33"? Esto se llama opcodes. En realidad nos interesa lo que viene después de ellos',
```

File: i18n/locales/es.ts
Comment (jmunoz94) [inline i18n/locales/es.ts:168]: Untranslated. See line 143. Also applies to 152.

File: i18n/locales/es.ts
Comment (jmunoz94) [inline i18n/locales/es.ts:173]:
```suggestion
        'Ahora solo queda decodificarlo, igual que hicimos en el ejercicio anterior',
```

**Review by jmunoz94 — COMMENTED** [review]
I reviewed half+ of the translations (the person who said that would review this can double-check and finish the review). Looks good overall, but there are some weird sentences here and there and "usted" and "tú" are unnecessarily mixed.

File: i18n/locales/es.ts
Comment (SuperAtic) [reply i18n/locales/es.ts:17]: Yeah, that's what I'm using. How you'd say it otherwise?

Comment (jmunoz94) [reply i18n/locales/es.ts]: Maybe "De momento se ve bien" or "De momento parece bien", but "tiene buena pinta" is understandable so I assume that one is also OK.

**Review by benalleng — CHANGES_REQUESTED** [review]
Looks like you accidentally forgot to export your translation file
should be a quick fix!
`/i18n/locales/index.ts`
``` suggestion
import en from './en'
import es from './es'
import nl from './nl'

const locales = {
  en,
  nl,
}

export default locales

```

**Comment by SuperAtic** [comment]
Too easy. Should be fine now, thanks for the support, and looking forward to seeing it merged.

**Review by afterburn — APPROVED** [review]
lgtm, and thanks for your contribution @SuperAtic

**Comment by SuperAtic** [comment]
This has been great guys and thanks so much for building this interactive-book(?) Not sure how you define it... Anyway, the process for the translation has been much easier than others we are working on, mostly because the team is reactive and attent to details 💪. We also published a [dedicated page to the book in our site](https://veintiuno.world/libro/salvando-satoshi/), if there's any content you'd like to add/remove, please DM us directly. Looking forward to translating the following chapters.

**Comment by ecurrencyhodler** [comment]
Thank YOU @SuperAtic for contributing! We really appreciate it.

I saw on your website that you show a catalogue of all the images. It might be a good idea not to show all of them. Reason being that the images will get periodically updated. But it's your call if you want to try and maintain the images as we keep developing our project.

**Pull request merged**
